src/main.py

```diff
@@ -1,60 +1,79 @@
 from flask import Flask, jsonify, request
-from flask_pymongo import PyMongo
+from pymongo import MongoClient
+from form_collection import FormCollection
 
 
 app = Flask(__name__)
 
+form_collection = FormCollection()
 
-#Running mongoDB on local host. Requires mongoDB to be installed and run locally. Must be generalized/dockerized
-app.config['MONGO_DBNAME'] = 'form'
-app.config['MONGO_URI'] = 'mongodb://localhost:27017/form'
 
-mongo = PyMongo(app)
+@app.route('/', methods=['GET'])
+def get_all_forms():
+    forms = form_collection.get_all_forms()
 
+    for form in forms:
+        del form['_id']
```

Comment:
I think deleting the object ids is a loss of valuable information. Rather convert the ObjectIDs to their string representation before returning. Then the user can utuluze the unique identifier 
` form['_id'] =  str(form['_id']) `

Reply:
This should be done in the form_collection so that the main file don't need to concern about database implementation details at all.